CMakeLists.txt

```diff
@@ -56,11 +57,14 @@ target_link_libraries (clio
   PUBLIC Boost::system
   PUBLIC Boost::log
   PUBLIC Boost::log_setup
+  PUBLIC Boost::stacktrace_backtrace
   PUBLIC cassandra-cpp-driver::cassandra-cpp-driver
   PUBLIC fmt::fmt
   PUBLIC OpenSSL::Crypto
   PUBLIC OpenSSL::SSL
   PUBLIC xrpl::libxrpl
+  PUBLIC dl
+  PUBLIC libbacktrace::libbacktrace
```

Comment:
Do we still need these two even with Boost.Stacktrace?

Reply:
Yes, Boost.stacktrace requires these two. https://www.boost.org/doc/libs/1_83_0/doc/html/stacktrace/configuration_and_build.html